Symfony/CS/ConfigInterface.php

```diff
@@ -128,4 +137,13 @@ public function getCacheFile();
      * @return string|null
      */
     public function getPhpExecutable();
+
+    /**
+     * Set the fixers to be used.
+     *
+     * @param FixerInterface[] $fixers
```

Comment:
`ConfigurationResolver` set a `FixerInterface[]` here, but in `.php_cs` we set a `string[]` here...